cert.go

```diff
@@ -386,6 +386,19 @@ func (c *Certificate) MarshalPEM() (pem_block []byte, err error) {
 	return io.ReadAll(asAnyBio(bio))
 }
 
+// MarshalDER converts the X509 certificate to DER-encoded format
+func (c *Certificate) MarshalDER() (der_block []byte, err error) {
```

Comment:
nit: don't need named return values here

Reply:
Agreed, but this matches the current codebase style